pkg/dns/provider/entry.go

```diff
@@ -282,6 +290,20 @@ func (this *Entry) Update(logger logger.LogContext, object *dnsutils.DNSEntryObj
 	return reconcile.UpdateStatus(logger, mod.Update())
 }
 
+func (this *Entry) setTTLStatus(spec *api.DNSEntrySpec, status *api.DNSEntryStatus, mod *resources.ModificationState, defaultTtl int64) {
```

Comment:
Instead I would calculate the effective TTL according to default and spec.
step1 : If this one is different from the actual setting of the entry it must be modified.
step 2. If this one is different from the actual status. the status must be updated